styles/mixins/patterns/_spinners.scss

```diff
@@ -15,6 +17,18 @@
   }
   color: var.$color-grey-6;
   // Assure that the spinner SVG is sized proportinally to local font-size
-  width: $-size;
-  height: $-size;
+  font-size: $-size;
+}
+
+/**
+ * Style a container with an svg.&__spinner as a full-screen loading spinner,
+ * by applying an overlay to the container and fixed-centered positioning to
+ * the contained svg icon
+ */
+@mixin full-screen-spinner() {
+  @include layout.overlay($variant: 'light');
```

Comment:
The implementation of a full-screen spinner on LMS uses a light background, and I do think it looks nicer than a heavy, dark one.